lib_eio/net.mli

```diff
@@ -187,8 +191,12 @@ val accept_sub :
 
 (** {2 Datagram Sockets} *)
 
-val datagram_socket : sw:Switch.t -> #t -> Sockaddr.datagram -> <datagram_socket; Flow.close>
-(** [datagram_socket ~sw t addr] creates a new datagram socket that data can be sent to
+val datagram_socket :
+     sw:Switch.t
+  -> #t
+  -> [Sockaddr.datagram | `UdpV4 | `UdpV6]
+  -> <datagram_socket; Flow.close>
+(** [datagram_socket ?addr ~sw t] creates a new datagram socket that data can be sent to
```

Comment:
```suggestion
(** [datagram_socket ~sw addr t] creates a new datagram socket that data can be sent to
```
Needs to explain what `addr` is for (source address? destination?)

Reply:
Done.